### PR TITLE
Port solidly to v2

### PR DESCRIPTION
**Pull request by mayorcoded**

### CODE REVIEW

File: src/vault/v2/adapter/solidly/SolidlyAdapter.sol [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    function _withdrawLP(uint256 amount) internal override {
+        gauge.withdraw(amount);
+    }
+
+    /*//////////////////////////////////////////////////////////////
+                            CLAIM LOGIC
+    //////////////////////////////////////////////////////////////*/
+    /// @notice Claim rewards from the Solidly gauge
+    function _claim() internal override {
+        try gauge.getReward(address(this), _getRewardTokens()) {
```
Comment (RedVeil) [inline src/vault/v2/adapter/solidly/SolidlyAdapter.sol:97]: lets just use `rewardTokens` here without defining a `_getRewardTokens()` function. If the creator doesnt want all the rewardTokens its up to them

Comment (RedVeil) [reply src/vault/v2/adapter/solidly/SolidlyAdapter.sol:97]: ok that makes sense. but than lets make it dynamic as a for loop. Cause potentially there arent 3 rewardsToken configured

Comment (RedVeil) [reply src/vault/v2/adapter/solidly/SolidlyAdapter.sol:97]: and could you add a comment explaining this in the function?

Comment (mayorcoded) [reply src/vault/v2/adapter/solidly/SolidlyAdapter.sol:97]: Alright, I will do that.

**Pull request merged**